app/service/rpc_service.ts

```diff
@@ -44,7 +44,16 @@ class RpcService {
     return false;
   }
 
-  getBytestreamUrl(bytestreamURL: string, invocationId: string, { filename = "", zip = "" } = {}): string {
+  getDownloadUrl(params: Record<string, string>): string {
+    return `/file/download?${new URLSearchParams(params)}`;
+  }
+
+  getBytestreamUrl(
```

Comment:
Can the `additionalParams` be removed from this func and below?

Reply:
Yup, done.